core/src/test/scala/akka/persistence/cassandra/session/javadsl/CassandraSessionSpec.scala

```diff
@@ -7,7 +7,7 @@ package akka.persistence.cassandra.session.javadsl
 import java.util.Optional
 
 import akka.Done
-import akka.cassandra.session.javadsl.CassandraSession
+import akka.cassandra.session.javadsl
```

Comment:
should this test be moved to the session project, or does it have anything specific to the plugin? maybe separate PR?

Reply:
Yes, that's in the workings.